x/oracle/abci_test.go

```diff
@@ -116,7 +116,7 @@ func createVotes(hash string, val sdk.ValAddress, rates sdk.DecCoins, blockHeigh
 
 func (s *IntegrationTestSuite) TestEndBlockerVoteThreshold() {
 	app, ctx := s.app, s.ctx
-	ctx = ctx.WithBlockHeight(1)
```

Comment:
Why did we have to update it to block 0 here?

Reply:
This just makes the math work out so that the prevote and vote are in the same voting period (decreasing from 5 to 3 made it so it synchronizes like that throughout the test starting at block 0)